src/org/opendatakit/briefcase/model/BriefcasePreferences.java

```diff
@@ -112,6 +114,16 @@ public static String getBriefcaseDirectoryProperty() {
     return Preference.APPLICATION_SCOPED.get(BriefcasePreferences.BRIEFCASE_DIR_PROPERTY,
         System.getProperty("user.home"));
   }
+
+  public static void setBriefcaseProxyProperty(HttpHost value) {
+    if (value == null) {
+	  Preference.APPLICATION_SCOPED.remove(BRIEFCASE_PROXY_HOST_PROPERTY);
```

Comment:
Should be spaces, not tabs. I think there are a few more places where tabs are introduced. Would be great to do a pass replacing them!